services/web/server/src/simcore_service_webserver/catalog_api_handlers.py

```diff
@@ -38,6 +34,15 @@ def json_dumps(v) -> str:
 
 VX = f"/{api_version_prefix}"
 
+
+# SEE https://pydantic-docs.helpmanual.io/usage/exporting_models/#modeldict
+RESPONSE_MODEL_POLICY = {
+    "by_alias": True,
+    "exclude_unset": False,
+    "exclude_defaults": False,
+    "exclude_none": True,
```

Comment:
exclude_none is a dangerous policy I think. what if one sets a nullable field as None (with default to something else). then it gets lost.

Reply:
changed